### PR TITLE
Add selu sweep

### PR DESCRIPTION
**Pull request by npetrovic-tenstorrent**
Add new script for sweep:
- selu

### CODE REVIEW

**Review by jdesousa-TT — APPROVED** [review]

File: tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```
Comment (TT-BrianLiu) [inline tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py:2]: Please fix

**Review by TT-BrianLiu — APPROVED** [review]

File: tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    (
+        [(3, 2, 192, 32)],
+        [ttnn.bfloat16],
+        [ttnn.TILE_LAYOUT],
+        [ttnn.DRAM_MEMORY_CONFIG],
+        ttnn.L1_MEMORY_CONFIG,
+        6861134,
+    ),
+    (
+        [(12, 224, 224)],
```
Comment (rfurko-tt) [inline tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py:58]: Could you please add tests, where last two dimensions are not divisible by 32? Thanks in advance!

File: tests/sweep_framework/sweeps/eltwise/unary/selu/selu.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+# Override the default timeout in seconds for hang detection.
+TIMEOUT = 30
+
+random.seed(0)
+
+# Parameters provided to the test vector generator are defined here.
+# They are defined as dict-type suites that contain the arguments to the run function as keys, and lists of possible inputs as values.
+# Each suite has a key name (in this case "suite_1") which will associate the test vectors to this specific suite of inputs.
+# Developers can create their own generator functions and pass them to the parameters as inputs.
+parameters = {
```
Comment (rfurko-tt) [inline tests/sweep_framework/sweeps/eltwise/unary/selu/selu.py:26]: where this `parameters` variable is used?

**Review by rfurko-tt — COMMENTED** [review]
Overall looks good
Please add a few more tests and explain where `parameters` variable is used. Once done, ready to merge.

File: tests/sweep_framework/sweeps/eltwise/unary/selu/selu.py (hunk 3 above)
Comment (npetrovic-tenstorrent) [reply tests/sweep_framework/sweeps/eltwise/unary/selu/selu.py:26]: Parameters are used in sweep generation phase, before sweep execution by another script - sweeps_parameter_genrator:
python3 tests/sweep_framework/sweeps_parameter_generator.py --elastic cloud --module-name eltwise.unary.selu.selu

**Review by rfurko-tt — APPROVED** [review]
LGTM

File: tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import pytest
+import torch
+import ttnn
+import traceback
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from tests.ttnn.python_api_testing.sweep_tests import ttnn_ops
+
+
+def run_eltwise_exp_tests(
```
Comment (eyonland) [inline tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py:16]: Please add the GH Issue number this tests is exposing as a failure in a comment.

**Review by eyonland — APPROVED** [review]
Thank you!!!!

**Pull request merged**